__tests__/app.test.js

```diff
@@ -62,6 +62,20 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200: Responds with an array in descending order of date created", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect([
```

Comment:
You should be passing the articles from the response here and using toBeSorted as the matcher for that.